contributing

```diff
@@ -0,0 +1,32 @@
+# Contributing to Project Open Data
```

Comment:
I believe I made that change earlier. https://github.com/cfpb/project-open-source/commit/adbb0af950f5c93e0970fe00d1625cc3d0f48b4e